package.json

```diff
@@ -67,6 +66,7 @@
   },
   "dependencies": {
     "react": "^16.9.0",
+    "fhir.js": "0.0.20",
```

Comment:
fhir.js is bundled into esm-api as a build-time module. The `dependencies` section should only be used for in-browser dependencies in the import map.

So this change is incorrect.

Reply:
Wait is this for typescript errors? That would be unfortunate.

The way that packmap works for packaging offline things is that it assumes all `dependencies` are in-browser dependencies. Could we perhaps use [`@types/fhir`](https://unpkg.com/browse/@types/fhir@0.0.30/) instead to solve this? Packmap knows to skip all the `@types` packages in the import map

Reply:
Funny thing is the build works okay by itself, but once you install it as node module there are typescript errors and `fhir.js` not found.

Reply:
Yeah dependencies are the only thing installed when another package does `npm install @openmrs/esm-api`.

Can you confirm if adding `@types/fhir.js` as a dependency works for you? See https://www.npmjs.com/package/@types/fhir.

To test, you can use `npm link`. See https://docs.npmjs.com/cli/link.html

Reply:
works now after adding `@types/fhir`